azure-pipelines.yml

```diff
@@ -82,3 +86,75 @@ stages:
         arguments: --build-arg IMAGE_TAG=1809
         tags: |
           1809
+    #------------------------------------------------
+    # Push images (push to default branch only)
+    #------------------------------------------------
+    - task: Docker@2
+      condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+      displayName: Login to Docker Hub
+      inputs:
+        command: login
+        containerRegistry: DockerHub
+    - task: Docker@2
+      condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+      displayName: Push webkitdev/base:1809 image
+      inputs:
+        command: push
+        containerRegistry: DockerHub
+        repository: webkitdev/base
+        tags: |
+          1809
+    - task: Docker@2
+      condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+      displayName: Push webkitdev/scripts:1809 image
+      inputs:
+        command: push
+        containerRegistry: DockerHub
+        repository: webkitdev/scripts
+        tags: |
+          1809
+    - task: Docker@2
+      condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+      displayName: Push webkitdev/scm:1809 image
+      inputs:
+        command: push
+        containerRegistry: DockerHub
+        repository: webkitdev/scm
+        tags: |
+          1809
+    - task: Docker@2
+      condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+      displayName: Push webkitdev/tools:1809 image
+      inputs:
+        command: push
+        containerRegistry: DockerHub
+        repository: webkitdev/tools
+        tags: |
+          1809
+    - task: Docker@2
+      condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+      displayName: Push webkitdev/msbuild-2022:1809 image
```

Comment:
Since we (will) use it downstream, can we also push the msbuild-2019 image to make sure we get an up-to-date-ish one when building the 1809 playstation ones?